Makefile

```diff
@@ -1,20 +1,27 @@
 all: build
 
 APP_NAME=cluster-monitoring-operator
+BIN=operator
```

Comment:
What do we need `BIN` for, if it is only used once?

Reply:
well, we currently use it twice, but we should actually use it in three places. I will update the third use in the `run` target